cmd/configs.go

```diff
@@ -0,0 +1,14 @@
+package cmd
+
+const base_url = "http://dspace.amritanet.edu:8080"
+
+const course_url = base_url + "/xmlui/handle/123456789/"
+
+const code_btech = "150"
+const code_ba_communication = "893"
+const code_ma_communication = "894"
+const code_integrated_msc_ma = "903"
+const code_mca = "331"
+const code_msw = "393"
+const code_mtech = "279"
```

Comment:
nit: Best practices. https://go.dev/doc/effective_go#constants


```suggestion
const (
	code_btech             = "150"
	code_ba_communication  = "893"
	code_ma_communication  = "894"
	code_integrated_msc_ma = "903"
	code_mca               = "331"
	code_msw               = "393"
	code_mtech             = "279"
)
```